src/fe-small-advertisements/src/features/person/personSlice.ts

```diff
@@ -0,0 +1,14 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+const personSlice = createSlice({
+  name: 'person',
+  initialState: 'LeBron James',
+  reducers: {
+    setName: (state, { payload }: PayloadAction<{ name: string }>) =>
+      payload.name,
+  },
+});
+
+export const { setName } = personSlice.actions;
+
+export default personSlice.reducer;
```

Comment:
🚓 
```suggestion
export const personReducer = personSlice;
```
This is a good oportunity, I believe, to create an issue to configure the eslint in such way so that when we export default an eslint throws an error?

Reply:
We would like this rule just for the frontEnd, or in the whole project?

Reply:
I added to the whole project.